ui/src/adapters/api/credentials.ts

```diff
@@ -65,7 +65,7 @@ export async function credentialIssue({
 }): Promise<APIResponse<Credential>> {
   try {
     const response = await axios({
-      baseURL: env.api.url,
+      baseURL: `${env.api.url}/${API_VERSION}`,
```

Comment:
I thought we were going to follow Jordi's advice and hardcode this. Why now have a variable for something that does not change? Can we add this suffix globally when we parse the env so it's part of the URL and we don't need to add it everywhere?

Reply:
nah, it's ok as it is.